Tribler/Core/defaults.py

```diff
@@ -163,6 +164,26 @@
 sessdefaults['http_api']['enabled'] = False
 sessdefaults['http_api']['port'] = -1
 
+# Credit mining config
+sessdefaults['credit_mining'] = OrderedDict()
+sessdefaults['credit_mining']['enabled'] = True
```

Comment:
Maybe we should disable it by default and make a toggle in the settings pane to enable it? Just a suggestion.